.circleci/config.yml

```diff
@@ -69,26 +69,24 @@ jobs:
           command: cargo login $cargo_TOKEN
       - run:
           name: setup git config
-          command: git config user.email "dev@mazzaroth.io" && git config user.name "CricleCI" && git config --global core.editor "\usr\local\bin\subl -n -w"
+          command: git config user.email "dev@mazzaroth.io" && git config user.name "CircleCI" && git config --global core.editor "\usr\local\bin\subl -n -w"
       - run:
           name: commit updates
           command: git add -A && git commit -m "Dependency/Version update for $CIRCLE_BRANCH"
-      - run:
-          name: pull latest from master
-          command: git pull --no-edit -Xours --rebase origin master
-      - run:
-          name: update master
-          command: git checkout master && git rebase -Xtheirs $CIRCLE_BRANCH && git push origin master
       - run:
           name: cargo publish
           command: |
             cargo publish --manifest-path xdr-rs-serialize-derive/Cargo.toml
             sleep 10
```

Comment:
Why is there a sleep here?

Reply:
You have to publish the macro repo first and without it being up on cargo, the second publish of the main repository will fail.